app/assets/stylesheets/battles.scss

```diff
@@ -6,6 +6,51 @@
   display: none;
 }
 
+//our additions changes
+
+body {
+  background-color: #2F3F73;
+}
+
+.color-0 {
+  background-color: #BFCCC9;
+}
+
+.color-1 {
+  background-color: #EBFBF8;
+}
+
 .avatar {
-  width: 50px;
+  margin-top: 10px;
+	margin-bottom: 5px;
+	border-radius: 50%;
```

Comment:
@J3RN I thought that might happen somewhere.  So apparently when I changed the brackets setting, it only made tabs look like 2 spaces even though they were still 4.  I had to change a setting to make them actually act as 2 spaces.  So those further tabbed lines are from before I fixed that.